node/main-support.ts

```diff
@@ -133,6 +133,19 @@ function getDurationDisplay(numOfSec: number): string {
   }
 }
 
+/**
+ * Generate frame rate display formatted as XXfps
+ * @param frameRate
+ */
+ function getFrameRateDisplay(frameRate: number): string {
```

Comment:
I think this is better done in the _template_ (`.html` file) 👍 

The reason some other values are stored in the `ImageElement` object rather than live-computed is because their computation is much more computationally heavy, while this one is just appending text 👍